doc-src/templates/default/fulldoc/html/css/common.css

```diff
@@ -0,0 +1,13 @@
+/* Default color is 888, change to 666 for at least a 4.5 contrast ratio */
+li .item { color: #666; cursor: pointer; }
+#search { width: 170px; position: static; margin: 3px; margin-left: 10px; font-size: 0.9em; color: #666; padding-left: 0; padding-right: 24px; }
```

Comment:
Is specifying width in px going to cause a problem with different screen sizes/layouts?

Reply:
Probably, but this is also a direct copy of the existing styling, but with a color change.

Reply:
I think it's just for the search pop up anyway. So even smaller layouts will probably have search that's this wide.